setup.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+from setuptools.command.install import install
+from shutil import copy2
+import os
+
+
+class RegisterGitPlugin(install):
+
+    def run(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        src = os.path.join(dir_path, "git-coverage")
+        dst = "/usr/bin/git-coverage"
```

Comment:
This doesn't appear to be venv-safe?  How should this work if you're installing in a venv?